wazuh/wazuh-dashboard
# Change default value of useExpandedHeader to false

### Description

The default value of the useExpandedHeader is changed to false so that the default black bar is hidden and to display it you have to use the setting

### Issues Resolved

- #241 

### Test

1. Remove the useExpandedHeader setting from opensearch_dashboard.yml file 
2. See that the black header bar does not appear.
3. Add the useExpandedHeader setting with the value true
4. See that the black header bar appear.

```
opensearchDashboards.branding:
  useExpandedHeader: true
```

### Check List
